exercises/proverb/.meta/src/reference/java/Proverb.java

```diff
@@ -0,0 +1,23 @@
+final class Proverb {
+    private String[] words;
+
+    Proverb(String[] words) {
+        this.words = words;
+    }
+
+    String recite() {
+        if (words.length < 1) return "";
+        String initWord = "",
+               result   = "";
+        for(int i = 0; i < words.length; i++) {
+            String word = words[i];
+            if (initWord.isEmpty()) {
+                initWord = word;
+            } else {
+                result += "the " + word + " was lost.\n";
+            }
+            if (i + 1 != words.length) result += "For want of a " + word + " ";
```

Comment:
This should have curly brackets around it, i.e.
```
if (i + 1 != words.length) {
  result += "For want of a " + word + " ";
}
```
It's considered best practice to always put curly brackets around if statements, even when they're only one line, to prevent errors if someone tried to add another line to it. I admit errors are less likely when the body of the if is on the same line as the if statement, but I still think it's worth using curly brackets just in case :)